src/ext/oblivc/copy.c

```diff
@@ -193,18 +172,6 @@ void ocShareMuxes(ProtocolDesc* pd,char* z,
   { ctx->recver.recv(ctx->recver.recver,tr,c,n,eltsize);
     ctx->sender.send(ctx->sender.sender,t,t2,n,eltsize);
   }
-#else
-  struct CorrFunXorArgs a = {.len=eltsize,.mask=t2};
-  if(protoCurrentParty(pd)==1)
-  { genrGateToShare(pd,ctx,t,t2,wc,n,eltsize);
-    //honestCorrelatedOTExtSend1Of2(ctx->sender,t,t3,n,eltsize,corrFunXor,&a);
-    honestCorrelatedOTExtRecv1Of2(ctx->recver,tr,c,n,eltsize);
-  }else
-  { evalGateToShare(pd,ctx,tr,wc,n,eltsize);
-    //honestCorrelatedOTExtRecv1Of2(ctx->recver,tr,c,n,eltsize);
-    honestCorrelatedOTExtSend1Of2(ctx->sender,t,t3,n,eltsize,corrFunXor,&a);
```

Comment:
Given that right now I don't know of any major users of these xor shares, I am okay with introducing a bandwidth regression here. Just file a bug about this saying that a new COTsender/COTrecver needs to be implemented for efficient shares.